ui/selectors/selectors.js

```diff
@@ -619,6 +627,34 @@ export function getGasIsLoading(state) {
   return state.appState.gasIsLoading;
 }
 
+/**
+ * Retrieves user preference to never see the "Switched Network" toast
+ *
+ * @param state - Redux state object.
+ * @returns Boolean preference value
+ */
+export function getNeverShowSwitchedNetworkMessage(state) {
```

Comment:
Nit: Would be better to have unit tests for all new functions; these two don't seem to have any.